MapboxNavigation/RouteMapViewController.swift

```diff
@@ -517,21 +542,20 @@ extension RouteMapViewController: RoutePageViewControllerDelegate {
         maneuverViewController.shieldImage = nil
         updateShield(for: maneuverViewController)
         
-        let streetLabelWidth = maneuverViewController.streetLabel.bounds.width
+        let streetLabelBounds = maneuverViewController.streetLabel.bounds
         let streetLabelFont = maneuverViewController.streetLabel.font!
 
         if let name = step?.names?.first {
-            maneuverViewController.streetLabel.text = name.abbreviated(width: streetLabelWidth, font: streetLabelFont)
+            maneuverViewController.streetLabel.text = name.abbreviated(bounds: streetLabelBounds, font: streetLabelFont)
         } else if let destinations = step?.destinations?.joined(separator: "\n") {
-            maneuverViewController.streetLabel.text = destinations.abbreviated(width: streetLabelWidth, font: streetLabelFont)
+            maneuverViewController.streetLabel.text = destinations.abbreviated(bounds: streetLabelBounds, font: streetLabelFont)
         } else if let step = step {
-            maneuverViewController.streetLabel.text = routeStepFormatter.string(for: step)?.abbreviated(width: streetLabelWidth, font: streetLabelFont)
+            maneuverViewController.streetLabel.text = routeStepFormatter.string(for: step)?.abbreviated(bounds: streetLabelBounds, font: streetLabelFont)
         }
-        maneuverViewController.distanceLabel.text = step!.distance > 0 ? distanceFormatter.string(from: step!.distance) : ""
+        maneuverViewController.distanceText = step!.distance > 0 ? distanceFormatter.string(from: step!.distance) : nil
```

Comment:
It would be cleaner to replace the `distanceText` property with a `distance` property whose `didSet` takes care of formatting the numeric distance into a string.